src/resources/SearchUsageMetrics/RawMetrics/RawMetricsInterface.ts

```diff
@@ -0,0 +1,136 @@
+import {SearchHubRawMetrics} from '../../Enums';
+
+export interface DailyRawMetricParameters extends CommonMetricParameters {
+    /**
+     * The last day to report for
+     * Format YYYY-MM-DD
+     */
+    to: string;
+
+    /**
+     * The first day to report for
+     * Format YYYY-MM-DD
+     */
+    from: string;
+}
+
+export interface MonthlyRawMetricParameters extends CommonMetricParameters {
+    /**
+     * The last month to report for
+     * Format YYYY-MM
+     */
+    to: string;
+
+    /**
+     * The first month to report for
+     * Format YYYY-MM
+     */
+    from: string;
+}
+
+export interface MonthlyRawMetricsParameters {
+    /**
+     * The month to report for
+     * Format YYYY-MM
+     */
+    month: string;
+
+    /**
+     * The minimum number of queries required for a search hub to be listed
+     */
+    minimumQueries?: string;
```

Comment:
Good question, in swagger, it's a string (https://platformdev.cloud.coveo.com/docs?urls.primaryName=Search%20Usage%20Metrics#/Raw%20Metrics/getRawMetricsForMonth), but maybe if pass a number instead, it will convert correctly?

Reply:
Indeed, its seems like we expect to receive a string and then we convert it to a number 🤔  Worst case, change it to a number and inside the client, convert the number to a string?